pkg/beholder/config.go

```diff
@@ -30,8 +31,14 @@ type Config struct {
 	// Batch processing is enabled by default
 	// Disable it only for testing
 	LogBatchProcessor bool
+
+	CSAAuthEnabled bool
```

Comment:
Is an explicit bool necessary? Or could we infer this from the presence of the other fields?